spec/app/rails_5/config/initializers/new_framework_defaults.rb

```diff
@@ -17,8 +17,9 @@
 # Require `belongs_to` associations by default. Previous versions had false.
 Rails.application.config.active_record.belongs_to_required_by_default = true
 
+# Removed in rails 5.2+
```

Comment:
This was needed only for tests for Rails 5.0. There is a separate 5.2 folder. So this change should not be required?

Reply:
It blows up on `rake spec`. That must run the wrong combination of directories and versions? I had trouble getting appraisal to work here.

Reply:
Hmm, made some changes. Lets see if build fails.

Reply:
Totally happy if it's not needed.